sdk/storage/azure-storage-files-shares/src/share_file_client.cpp

```diff
@@ -641,8 +641,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
     int64_t fileRangeSize;
     if (firstChunkOptions.Range.HasValue())
     {
-      fileSize = std::stoll(firstChunk->ContentRange.GetValue().substr(
-          firstChunk->ContentRange.GetValue().find('/') + 1));
+      fileSize = firstChunk->ContentLength;
```

Comment:
What is `ContentLength`? Is it the whole file size or just the chunk we downloaded in this request? Would it be clearer if we name it `FileSize`?

Reply:
Renamed.